Dockerfile

```diff
@@ -9,14 +9,14 @@
 
 # Similarly, the BUILD_IMAGE argument can be overwritten
 # but this is generally not needed.
-ARG BUILD_IMAGE=gradle:6.2.1-jdk8
+ARG BUILD_IMAGE=gradle:6.9-jdk8
 
 #
 # Build phase: Use the gradle image for building.
 #
 FROM ${BUILD_IMAGE} as build
 USER root
-RUN apt-get update -qq && DEBIAN_FRONTEND=noninteractive apt-get install -y -qq zeroc-ice-all-runtime libblosc1
+RUN apt-get update -qq && DEBIAN_FRONTEND=noninteractive apt-get install -y -qq libzeroc-ice3.7-java libblosc1
```

Comment:
I'm very much struggling to remember why zeroc would be needed here. It's almost certainly a copy-n-paste error and removing it works for me locally.